ksp_plugin/manœuvre.hpp

```diff
@@ -76,6 +78,9 @@ class Manœuvre {
   // Intensity and timing must have been set.
   Instant final_time() const;
 
+  // Intensity and timing must have been set.
```

Comment:
Returns true if and only if |this| fits between begin and end?
If that's not clear enough, the semantics have to be described as the one-liner implementation...
I really don't see how this can benefit from a comment.

On the other hand, this should be const.